src/native/clrmath/src/fwd/cbrtf.c

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "clrmath.h"
+
+float clrmath_cbrtf(float x)
```

Comment:
Is the C++ compiler able to optimize out these forwarders in all situations, and to always pick the local implementation? 

It feels fragile. It may be better if the C method names for the local implementations included a prefix.

Reply:
These forwarders only exist for the methods that are still coming from the C runtime (because they are implemented in assembly for aocl-libm-ose).
Since it's all statically linked, `COMSingle::Cbrt` is compiled down to call `cbrtf` directly.

Reply:
It assumes good link-time code generation that is not given, especially on some platforms targeted by Mono.

Reply:
I'll update these to just be `#define clrmath_cbrtf cbrtf.

That being said, I would be very surprised if LLVM failed to do the basic inlining optimization here.